code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -516,7 +506,7 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 
 /obj/item/roller/medevac
 	name = "medevac stretcher"
-	desc = "A collapsed medevac stretcher that can be carried around."
+	desc = "A collapsed medevac stretcher that can be carried around. Can be used to instantly transport a marine to a linked beacon. Don't forget the beacon! Right click to activate."
```

Comment:
I'm nitpicking but the right click to activate tip should be on the deployed version. Or something like 'use in hand to deploy then right click' here.

Reply:
I can't believe you'd pick my PR's